### PR TITLE
Backport session upload event to 5.0

### PR DESCRIPTION
**Pull request by quinqu**
Backport of a fix from master for bringing back session upload event

Original fix against master: [#5470](https://github.com/gravitational/teleport/pull/5470)

### CODE REVIEW

**Comment by awly** [comment]
@quinqu is this a backport of a fix from `master` or a standalone change?

**Comment by quinqu** [comment]
@awly it is a backport of a fix from master

File: lib/events/events.proto [hunk 1]
```diff
@@ -1143,6 +1143,7 @@ message OneOf {
         events.AppSessionStart AppSessionStart = 37;
         events.AppSessionChunk AppSessionChunk = 38;
         events.AppSessionRequest AppSessionRequest = 39;
+        events.SessionUpload SessionUpload = 40;
```
Comment (awly) [inline lib/events/events.proto:1146]: use the same field IDs as on `master`, to allow 5.x binaries to communicate with 6.x
```suggestion
        events.SessionUpload SessionUpload = 43;
```

**Review by awly — APPROVED** [review]

**Review by russjones — APPROVED** [review]
Bot.

**Comment by russjones** [comment]
@quinqu Please don't merge until March 1st.

**Review by a-palchikov — APPROVED** [review]

**Review by awly — APPROVED** [review]

**Pull request merged**